tests/io/v3/base/test_explore.py

```diff
@@ -127,6 +127,9 @@
 
 
 def test_search(api):
```

Comment:
Test method name is bit vague. Rename either test-method name or update docstring.

Reply:
@Mangusingh-te this method is testing the _search method of base class that is why we have name to method ```test_search```. do you have any better suggestion?